### PR TITLE
Flesh out IntelliJ workspace page

### PR DESCRIPTION
**Pull request by parlough**

### CODE REVIEW

File: source/plugin/workspace/idea.rst [hunk 1]
```diff
@@ -72,13 +81,42 @@ Importing your project
...
+
+Minecraft Dev Plugin
+====================
+
+The Minecraft Development plugin for IntelliJ is a great plugin by a community member which makes plugin project
+creation much easier while also providing some neat and useful features for development.
+
+Creating your project
+~~~~~~~~~~~~~~~~~~~~~
+* Open **IntelliJ IDEA**.
```
Comment (ST-DDT) [inline source/plugin/workspace/idea.rst]: Empty line after the title

**Review by ST-DDT — CHANGES_REQUESTED** [review]

File: source/plugin/workspace/idea.rst (hunk 1 above)
Comment (parlough) [reply source/plugin/workspace/idea.rst]: Oops, will do.

File: source/plugin/workspace/idea.rst [hunk 2]
```diff
@@ -72,13 +81,42 @@ Importing your project
...
+* Click ``Create New Project``.
+* Select ``Gradle`` in the popup.
```
Comment (DenWav) [inline source/plugin/workspace/idea.rst]: Should be "Minecraft Plugin".

Comment (parlough) [reply source/plugin/workspace/idea.rst]: Ah yeah, forgot to edit that after copy pasting. Thanks.

**Review by ST-DDT — APPROVED** [review]
Looks good to me. I have to check the preview later as well.

**Review by kashike — APPROVED** [review]

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/8350f9e/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/8350f9e/plugin/workspace/idea.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/703#issuecomment-402230980), not to the direct url to the preview._

**Review by ST-DDT — APPROVED** [review]

**Comment by Inscrutable** [comment]
Can I be picky and remind you that Titles should use Title Case?

**Comment by parlough** [comment]
@Inscrutable I'm not sure what Titles you are talking about? I matched the case of the existing subtitles.

**Comment by phit** [comment]
`Creating your project` should be [`Creating Your Project`](http://www.grammar-monster.com/lessons/capital_letters_title_case.htm)
https://docs.spongepowered.org/stable/en/contributing/spongedocs.html#style-guide

the other headings in the same file don't seem to follow the rules.. 

edit: I blame Inscrutable https://github.com/SpongePowered/SpongeDocs/pull/456 😆

**Comment by Inscrutable** [comment]
Yeah, those must have got by us in 2016. Oops. Probably too much in one PR. Anyway, you can fix them or I can poke it with the magic edity stick later. Otherwise this is good to go.

**Pull request merged**